crates/types/src/services/executor.rs

```diff
@@ -336,7 +311,7 @@ impl ExecutionKind {
         match self {
             ExecutionKind::DryRun => ExecutionTypes::DryRun(t),
             ExecutionKind::Production => ExecutionTypes::Production(t),
-            ExecutionKind::Validation => ExecutionTypes::Validation(t),
+            ExecutionKind::Validation => todo!("remove this variant"),
```

Comment:
If this `todo!` still supposed to be here? Or is this part of the follow-up PR to remove `ExecutionKind`?